app/controllers/admin/order_histories_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Admin::OrderHistoriesController < Admin::AdminController
+  before_action :load_orders, only: :index
+
+  def index
+    @pagy, @order_histories = pagy(@order.order_histories, items: Settings.DIGIT_5)
```

Comment:
```suggestion
    @pagy, @order_histories = pagy(@order.order_histories, items: Settings.DIGIT_5)
```
Viết 1 scope sort theo mới nhất nữa nha. Để tăng trải nghiệm người dùng hơn